src/ios/AKPlugin.m

```diff
@@ -60,8 +60,19 @@ - (void)loginWithEmail:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)getAccount:(CDVInvokedUrlCommand *)command {
+  if (_accountKit == nil) {
+      _accountKit = [[AKFAccountKit alloc] initWithResponseType:AKFResponseTypeAccessToken];
```

Comment:
I only did a quick fix to make it work for our use case. I have not done any thorough to see that this fix doesn't break other use cases. We have solved this temporarily and are using our own custom version for now.
I can see if I get a chance to test some other use cases.